### ISSUE TITLE
[Feature Request] Save state and settings in seperate files

### ISSUE THREAD

**Issue by tunbridgep** [giuspen/cherrytree#616]
So, currently cherrytree saves it's settings in config.cfg. This is fine. But it also stores things like the last-opened file history, cursor position, windows positions etc in the same file.

This is potentially "bad practice" (despite being standard on Windows) because it makes any sort of configuration management tedious and annoying. If I want cherrytree to work the same way on multiple PCs, and want to sync the config file (say, through github), I run into a problem where the config file is polluted with unnecessary changes. Even if I were to ignore these, it still runs into the problem of having one PC where, for example, the recent documents list points to invalid documents.

Would it be possible to split the config into two files, one for permanent settings changes (font size, behaviour, etc) and one for "history" (recently opened documents, cursor and windows positions, etc etc).

Many programs on Linux already do this (~/.bash_history, for example).

**Comment by giuspen** [comment]
Makes sense but may have to wait for the Gtkmm3 version to be implemented
